eureka/S4_generate_lightcurves/s4_genLC.py

```diff
@@ -165,14 +165,16 @@ def lcJWST(eventlabel, s3_meta=None):
                 # Correct for drift/jitter
                 for n in range(meta.n_int):
                     # Need to zero-out the weights of masked data
-                    weights = (~optspec[n].mask).astype(int)
+                    #weights = (~optspec[n].mask).astype(int)
```

Comment:
Replace this with the following:
`weights = (~np.ma.getmaskarray(optspec[n])).astype(int)`